components/RunsPage/CategoryInfo/CategoryInfo.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import type { CategoryViewModel } from '~/lib/api/data-contracts'
+
+interface CategoryInfoProps {
+  category: CategoryViewModel
+}
+
+defineProps<CategoryInfoProps>()
+</script>
+<template>
+  <div class="flex max-w-[330px] flex-col p-6">
+    <h2 class="text-lg font-semibold">{{ category.name }}</h2>
+    <h3 class="mt-6 font-medium">Run Type Description</h3>
+    <p class="text-xs">
+      {{ category.info || '-' }}
+    </p>
+    <h3 class="mt-6 font-medium">Run Type Rules</h3>
+    <p class="text-xs">
+      {{ category.info || '-' }}
+    </p>
```

Comment:
This doesn't really make sense. There is no distinction between description and rules in our current data model.